File: FE/SuperMarketAngular/src/app/models/Order.ts
import { ResourceProduct } from "./Resources/ProductResource";


export class Order {
  
    id: number = 0 ;
    total: number = 0 ; 
    address: string = "" ;
    listOfProducts: number = 0 ;
    products: ResourceProduct = new ResourceProduct() ;
    OrderProduct: number = 0 ;

    Products(){

    }
}

File: Order/Entities/Order.cs
using System;
using System.Collections.Generic;

namespace Order.Entities
{
    public class OrderEntity
    {
        public int id { get; set; }
        public double total { get; set; } = 0;
        public String address { get; set; }
        public String listOfProducts { get; set; }
        public List<ProductEntity> products { get; set; }
        public List<OrderProduct> OrderProduct { get; set; }

    }
}
